src/constants/permit.ts

```diff
@@ -0,0 +1,172 @@
+    '0xD67de0e0a0Fd7b15dC8348Bb9BE742F3c5850454': { type: PermitType.AMOUNT, version: '1' }, //BNB
+    '0xD0660cD418a64a1d44E9214ad8e459324D8157f1': { type: PermitType.AMOUNT, version: '1' }, //WOOFY
+    '0xd6070ae98b8069de6B494332d1A1a81B6179D960': { type: PermitType.AMOUNT, version: '1' }, //BIFI
+    '0x82f0B8B456c1A451378467398982d4834b6829c1': { type: PermitType.AMOUNT, version: '1' }, //MIM
+    '0x3A3841f5fa9f2c283EA567d5Aeea3Af022dD2262': { type: PermitType.AMOUNT, version: '1' }, //SHADE
+    '0x818ec0A7Fe18Ff94269904fCED6AE3DaE6d6dC0b': { type: PermitType.AMOUNT, version: '1' }, //ATRI
+    '0x627524d78B4fC840C887ffeC90563c7A42b671fD': { type: PermitType.AMOUNT, version: '1' }, //KEK
+    '0x9fC071cE771c7B27b7d9A57C32c0a84c18200F8a': { type: PermitType.AMOUNT, version: '1' }, //iFUSD
+    '0x195FE0c899434fB47Cd6c1A09ba9DA56A1Cca12C': { type: PermitType.AMOUNT, version: '1' }, //MUNNY
+    '0x47d0d625638b56084e76b8720475d175d171af9A': { type: PermitType.AMOUNT, version: '1' }, //PENNI
+    '0x05848B832E872d9eDd84AC5718D58f21fD9c9649': { type: PermitType.AMOUNT, version: '1' }, //STEAK
+    '0x6626c47c00F1D87902fc13EECfaC3ed06D5E8D8a': { type: PermitType.AMOUNT, version: '1' }, //WOO
+    '0x468003B688943977e6130F4F68F23aad939a1040': { type: PermitType.AMOUNT, version: '1' }, //SPELL
+    '0xbbc4A8d076F4B1888fec42581B6fc58d242CF2D5': { type: PermitType.AMOUNT, version: '1' }, //FONT
+    '0x6a545f9c64d8f7B957D8D2e6410B52095A9E6c29': { type: PermitType.AMOUNT, version: '1' }, //CFi
+    '0x3129662808bEC728a27Ab6a6b9AFd3cBacA8A43c': { type: PermitType.AMOUNT, version: '1' }, //DOLA
+    '0xb84527D59b6Ecb96F433029ECc890D4492C5dCe1': { type: PermitType.AMOUNT, version: '1' }, //INV
+    '0x3b57f3FeAaF1e8254ec680275Ee6E7727C7413c7': { type: PermitType.AMOUNT, version: '1' }, //EXOD
+    '0x5d5530eb3147152FE78d5C4bFEeDe054c8d1442A': { type: PermitType.AMOUNT, version: '1' }, //FEED
+    '0xEFF6FcfBc2383857Dd66ddf57effFC00d58b7d9D': { type: PermitType.AMOUNT, version: '1' }, //JulD
+    '0x5C4FDfc5233f935f20D2aDbA572F770c2E377Ab0': { type: PermitType.AMOUNT, version: '1' }, //HEC
+    '0x5602df4A94eB6C680190ACCFA2A475621E0ddBdc': { type: PermitType.AMOUNT, version: '1' }, //SPA
+    '0x9F47F313ACFd4bdC52F4373b493EaE7d5aC5b765': { type: PermitType.AMOUNT, version: '1' }, //JOE
+    '0xfa1FBb8Ef55A4855E5688C0eE13aC3f202486286': { type: PermitType.AMOUNT, version: '1' }, //FHM
+    '0x9879aBDea01a879644185341F7aF7d8343556B7a': { type: PermitType.AMOUNT, version: '1' }, //TUSD
+    '0x6496994241804D7fE2b032901931e03bCD82301F': { type: PermitType.AMOUNT, version: '1' }, //MODA
+    '0xB66b5D38E183De42F21e92aBcAF3c712dd5d6286': { type: PermitType.AMOUNT, version: '1' }, //PILLS
+    '0xfB98B335551a418cD0737375a2ea0ded62Ea213b': { type: PermitType.AMOUNT, version: '1' }, //miMATIC
+    '0xDDc0385169797937066bBd8EF409b5B3c0dFEB52': { type: PermitType.AMOUNT, version: '1' }, //wMEMO
+  },
+}
+
+export const EIP712_DOMAIN_TYPE = [
```

Comment:
duplicate `EIP712Domain`

Reply:
permit use many type of EIP712 domain, so better leave it a version here

Reply:
EIP712Domain vs EIP712_DOMAIN_TYPE are the same. Let's remove 1 of them